openllm-python/src/openllm/_service.py

```diff
@@ -69,6 +71,90 @@ async def generate_stream_v1(input_dict: dict[str, t.Any]) -> t.AsyncGenerator[s
   else:
     return runner.generate_iterator.async_stream(qa_inputs.prompt, adapter_name=qa_inputs.adapter_name, echo=echo, **qa_inputs.llm_config.model_dump())
 
+@svc.api(route='v1/completions', input=_JsonInput, output=bentoml.io.Text())
+async def completion_v1(input_dict: dict[str, t.Any], ctx: bentoml.Context):
+  stream = input_dict.pop('stream', False)        # Determine whether to stream the response
+  echo = input_dict.pop('echo', False)
+  input_dict.pop('model', None)     # We don't need the exact model name here
+  default_config = llm_config.model_dump(flatten=True)
+
+  for key in list(input_dict.keys()):
+    if key == 'prompt': continue
+    elif key == 'max_tokens':
+      default_config['max_new_tokens'] = input_dict[key]
+    elif key in default_config:
+      default_config[key] = input_dict[key]   # Replace the default config with the input config
+    del input_dict[key]                       # Remove the key from input_dict
+  input_dict['adapter_name'] = None
```

Comment:
I think we shouldn't do this. Probably best to have a function on the `LLMConfig` to convert to openai attributes or something.

Reply:
Yes that's right, we could do that.